docs/admin/templates/managing-templates/devcontainers/index.md

```diff
@@ -1,4 +1,4 @@
-# Dev containers
+# Envbuilder Dev Containers
```

Comment:
Why does it lead with `Envbuilder`?

Reply:
we need to decide what to do with this set of docs - I re-titled them and made some quick changes to sections to be envbuilder-specific
we aren't removing envbuilder with this PR, so I want to strip these down as much as possible, but don't want to remove and redirect the docs just yet.